src/content/docs/guides/upgrade-to-biome-v2.mdx

```diff
@@ -0,0 +1,236 @@
+            <PackageManagerCommand
+                npm="install --save-dev --save-exact @biomejs/biome@2.0.0"
+                pnpm="add --save-dev --save-exact @biomejs/biome@2.0.0"
+                yarn="add --dev --exact @biomejs/biome@2.0.0"
+                bun="add --dev --exact @biomejs/biome@2.0.0"
+                deno="add --dev npm:@biomejs/biome@2.0.0" />
+        </li>
+        <li>
+            Run the `migrate` command to update the configuration:
+            <PackageManagerBiomeCommand command="migrate --write" />
+        </li>
+        <li>
+            You're all set! The `migrate` command should have updated your configuration to *mitigate* possible [breaking changes](#breaking-changes).
+        </li>
+    </ol>
+</Steps>
+
+
+
+## Breaking changes
+
+While the project and team are committed to reducing the number of breaking changes, v2 comes with some breakages that are worth explaining. This section will walk through the most relevant ones, and it will explain the reasons of the breaking change and provide a solution, if applicable.
+
+### Rome-related features aren't supported anymore
+
+All features related to the old Rome project aren't supported anymore. If you still relied on those features, you must upgrade your project:
+- rename `rome.json` to `biome.json`.
+- rename `// rome-ignore` to `// biome-ignore`.
+- rename `ROME_BINARY` to `BIOME_BINARY`.
+- the suppression comment format `// biome-ignore lint(<GROUP_NAME>/<RULE_NAME>): <explanation>` is no longer supported. use `// biome-ignore lint/<GROUP_NAME>/<RULE_NAME>: <explanation>` instead.
+
+### The option `--config-path` is removed
+
+The CLI option `--config-path` is removed from the commands `biome lsp-proxy` and `biome start`.
+
+The option was overriding the configuration path for all workspaces opened in the Biome daemon, which led to a configuration mismatch problem when multiple projects are opened in some editors or IDEs.
+
+If you use an LSP-compatible editor, you can use its settings to define the configuration path of the project.
+
+<EditorSettings zed={{ name: "configuration_path", value: "\"./frontend/biome.json\""}} vsCode={{ name: "configurationPath", value: "\"./frontend/biome.json\"" }}></EditorSettings>
+
+If you are a developer of a plugin, please update your plugin to use the `workspace/configuration` response instead of using the `--config-path` argument. Biome’s LSP will resolve a configuration in the workspace automatically, so it is recommended to keep it empty unless you are using a custom configuration path.
+
+### The options `ignore` and `include` are removed, and replaced by `includes`
+
+If you ran the command `migrate` already, you shouldn't have regressions. The reason why these two fields were merged into a  new one  was because the underneath implementation of glob engine was flawed, causing some edge cases that we couldn't fix unless we changed our engine, and introduce breaking changes.
```

Comment:
```suggestion
If you ran the command `migrate` already, you shouldn't have regressions. The reason why these two fields were merged into a single new one was because the implementation of Biome’s old glob engine was flawed, causing some edge cases that we couldn't fix unless we changed our engine. Unfortunately, we couldn’t do this without breaking changes.
```